src/collective/solr/tests/test_query.py

```diff
@@ -173,7 +170,6 @@ def testSolrSpecifics(self):
 class QueryTests(TestCase):
 
     def setUp(self):
-        provideUtility(SolrConnectionConfig(), ISolrConnectionConfig)
```

Comment:
@tomgross @do3cc by removing SolrConnnectionConfig here and relying on a plone.app.registry lookup for the connection, we break all the unit tests here. We either have to mock SolrConnectionConfig for all those tests, or re-add the SolrConnectionConfig.